test/acceptance/README.md

```diff
@@ -75,3 +77,41 @@ used to deploy the environment, needs it.
 
 There is a tag for each feature file
 
+```
```

Comment:
How do you plan to keep this list updated? Carefull review in each PR adding new acceptance test?